### ISSUE TITLE
Archer's grappling hook collides with nothing

### ISSUE THREAD

**Issue by mugg91** [transhumandesign/kag-base#1305]
## Description

This is an online bug.

Archer's grapple will collide with Red Barrier and vanish.
After the Red Barrier disappears, there is a rare chance the grapple will still collide with the middle of the map and therefore be rendered unusable.
This has occured to me two times so far.

The second occurence: I joined Small CTF, the Red Barrier has already been gone. I played Knight for a bit, then switched to Archer. I wanted to grapple to the other side and noticed the grapple didn't show up so I fell to the depth. After respawning I confirmed that grapple shows up on the left side but collides with "nothing" and disappears in the middle of the map.

## Reproduction

There are no known reproduction steps.

**Comment by mugg91** [comment]
The bug still exists in the current build 4234.
I joined the on-going match, played for a while as Knight, switched team, switched to Archer, then I wanted to jump off somewhere trying to use grapple and notice it didn't come out so I died from falling too high.
After respawning and going to the mid again, I noticed the grapple was still not coming out, but when going back to our flag room it came out again.
